summarization/summarizer_main.py

```diff
@@ -250,18 +257,23 @@ def _get_attribute_for_contents(self, content):
             if content.content_type == ContentType.IMAGE:
                 media = content
                 stamp_descriptor_embedding = media.img_description_embedding
+
+            elif content.content_type == ContentType.QUOTE:
+                quote = content
+                stamp_descriptor_embedding = quote.embedding
             else:
                 embedded = content
                 stamp_descriptor_embedding \
                     = self._get_stamp_descriptor_for_embedded_content(content)
 
-        return text, media, embedded, stamp_descriptor_embedding
+        return text, media, embedded, quote, stamp_descriptor_embedding
```

Comment:
Similar to Contents(class) and ContentType(enum), we should have StampPage an StampPageType.
text, media, embedded, quote, stamp_descriptor_embedding all these values should be in StampPage.
I'm resolving it because of time contraints.